src/app/routes/women/women.component.scss

```diff
@@ -9,6 +9,7 @@ h1{
     text-decoration: underline;
     text-decoration-color: orange;
     text-underline-offset: 7px;
+    cursor: pointer;
```

Comment:
if same code applies for multiple component, you can write the css code in styles.scss file rather then rewriting same css code in every component